docs/whats_new.rst

```diff
@@ -45,6 +45,7 @@ Bug
 - Fixed RANSAC to give consistent results with a fixed seed across different chunk sizes, by `Austin Hurst`_ and `Yorguin Mantilla`_ (:gh:`43`)
 - Fixed "bad channel by flat" threshold in :meth:`NoisyChannels.find_bad_by_nan_flat` to be consistent with MATLAB PREP, by `Austin Hurst`_ (:gh:`60`)
 - Changed "bad channel by deviation" and "bad channel by correlation" detection code in :class:`NoisyChannels` to compute IQR and quantiles in the same manner as MATLAB, thus producing identical results to MATLAB PREP, by `Austin Hurst`_ (:gh:`57`)
+- Fixed array-reshaping bug that broke RANSAC completely, by `Austin Hurst`_ (:gh:`67`)
```

Comment:
Could you go into a tiny bit more detail here, please? I.e., **what** was the result of the reshaping bug, something like: due to the reshaping bug, the correlation between windows were not made out of directly consecutive samples, but samples with lots of space in between

(rephrase to be accurate / understandable: I know it's hard to do in such a short space, but a bit more detail will help down the line)

Reply:
yes, thanks!